lib/resolveHookfiles.js

```diff
@@ -7,9 +7,7 @@ const glob = require('glob');
 const basename = process.platform === 'win32' ? path.win32.basename : path.basename;
 
 
-function resolveGlob(pattern, workingDirectory = null) {
-  const cwd = workingDirectory || process.cwd();
-
+function resolveGlob(cwd, pattern) {
```

Comment:
I later realised that this doesn't always represent the current working directory and the name is maybe tricky.

Perhaps it would be clearer to rename this from `cwd` to something else, like `directory`, `base` or `working_directory`. "cwd" stands for current working directory which I don't believe this is.